src/import/ImportWallet.tsx

```diff
@@ -219,12 +221,12 @@ function ImportWallet({ navigation, route }: Props) {
                   shouldShowClipboard={shouldShowClipboard}
```

Comment:
The 'Paste' button isn't in the designs. Is it supposed to still show up?

Reply:
@jh2oman yes, designs updated to include this. Just need to update the paste button color to match design.